build/config.go

```diff
@@ -12,6 +12,13 @@ const (
 	callSiteLong  = "long"
 
 	defaultLogCompressor = Gzip
+
+	// DefaultMaxLogFiles is the default maximum number of log files to
+	// keep.
+	DefaultMaxLogFiles = 3
```

Comment:
If we touch these, I also feel like it would be a good idea to slightly increase those numbers... Maybe 10 and 20 respectively? It just seems to happen often that users don't have what we need in their logs anymore. Or perhaps we could make it dependent on the log level? If they're running with DBG or higher, increase the numbers? That would probably be out of scope for this PR, just brain storming a bit here.

Reply:
cool - updated the defaults in a commit at the end (just to keep this commit a refactor) :)